README.md

```diff
@@ -5,52 +5,68 @@
 
 ## Quickstart
 
-Before you can begin with Yari, you need [Content](https://github.com/mdn/content).
-See its README which basically, says something like this:
+Before you can start working with Yari, you need a copy of the MDN [Content](https://github.com/mdn/content)
+directory — this is the content Yari will render when we run it locally. The
+easiest way to get a local copy of this is to run the following command
+(although bear in mind that you might want to clone your own fork of the repo
+instead, for when you create pull requests against the content):
 
     git clone https://github.com/mdn/content.git mdn/content
```

Comment:
I'd be tempted to merge all of this. Something like:
----
Before you can start working with Yari, you need to:
1. Install git, Node.js (>= 12.0.0), and Yarn 1.
1. Fork the MDN [content](https://github.com/mdn/content.git) and [yari]() repositories using the Fork button on Github

   [Insert image of fork button in github here]
1. Clone (copy) the forked repositories to your computer using the command line instructions shown below replace `[your account]` with where you forked the repositories):
   ```sh
   git clone https://github.com/[your_account]/content.git
   git clone https://github.com/[your_account]/yari.git
   cd yari
   ```

Reply:
Agreed; this looks much better, so I'm stealing it, nearly verbatim. Thanks @hamishwillee !